generated/nifgen/session.py

```diff
@@ -3532,6 +3532,8 @@ def create_arb_sequence(self, waveform_handles_array, loop_counts_array):
         '''
         vi_ctype = _visatype.ViSession(self._vi)  # case S110
         sequence_length_ctype = _visatype.ViInt32(0 if waveform_handles_array is None else len(waveform_handles_array))  # case S160
+        if loop_counts_array is not None and len(loop_counts_array) != len(waveform_handles_array):  # case S160
```

Comment:
Looks like a small bugfix. Worth noting?

Reply:
I don't think we need to.